src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/clients/RoleAssignmentServiceApi.java

```diff
@@ -60,4 +61,11 @@ RoleAssignmentResource queryRoleAssignments(
         @RequestHeader(SERVICE_AUTHORIZATION) String s2sToken,
         @RequestBody MultipleQueryRequest queryRequest
     );
+
+    @GetMapping(
+        value = "/am/role-assignments/actors/{actorId}",
+        produces = "application/vnd.uk.gov.hmcts.role-assignment-service.get-assignments+json;charset=UTF-8;version=1.0"
+    )
+    RoleAssignmentResource getRolesByActorId(@PathVariable("actorId") String actorId,
+                                           @RequestHeader(SERVICE_AUTHORIZATION) String serviceAuthToken);
```

Comment:
This method is the same as getRolesForUser() could we re-use or remove one of them

Reply:
same answer as above comment, pls check this link:
https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1507735022&__ncforminfo=qf0H4EZL3OaVUokH7BLm6lX6WKXTQ7WU-e9MV-13vkDcHmGUuXzdBIvzMdU7onu6L26Jkv7mFKCLGzuj4CCcu1RJCw64Vdyq#Userrolesandruleconfigurationrequirements-2.Worktypefilter

Reply:
endpoint removed